db/migrate/20190724140835_create_books.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :title
+      t.string :category
+      t.boolean :read
```

Comment:
We need to track the percentage of the book that has already been read by the user. So it's not a simple `boolean` we need here. Probably we need to include an attribute that updates the page/chapter that's already read.

Reply:
Well, this is listed among optional functionality, not the requirements. I'm probably going to to add it later, but for now I at least want to get simple version working well.

Reply:
Same for the bookstore API mentioned in the beginning